src/units.cpp

```diff
@@ -226,9 +226,8 @@ bool updateUnitMultiplier(double &multiplier,
                 auto model = owningModel(units);
                 if (model != nullptr) {
                     auto refUnits = model->units(ref);
-                    if ((refUnits == nullptr) || (refUnits->isImport() == true)) {
-                        updated = false;
-                        return updated;
+                    if ((refUnits == nullptr) || (refUnits->isImport())) {
```

Comment:
I am being very pedantic here, but: no need for parentheses around `refUnits->isImport()`.

Reply:
Okay, has been changed.